iac-extensions/docker/src/main/java/org/sonar/iac/docker/checks/utils/CommandDetector.java

```diff
@@ -31,23 +31,17 @@
 import org.sonar.iac.common.api.tree.HasTextRange;
 import org.sonar.iac.common.api.tree.impl.TextRange;
 import org.sonar.iac.common.api.tree.impl.TextRanges;
-import org.sonar.iac.docker.checks.utils.command.IncludingUnresolvedArgumentsArgumentResolutionPredicate;
 import org.sonar.iac.docker.checks.utils.command.CommandPredicate;
+import org.sonar.iac.docker.checks.utils.command.CommandPredicate.Type;
+import org.sonar.iac.docker.checks.utils.command.IncludingUnresolvedArgumentsArgumentResolutionPredicate;
 import org.sonar.iac.docker.checks.utils.command.IncludingUnresolvedArgumentsPredicate;
-import org.sonar.iac.docker.checks.utils.command.MultipleUnorderedOptionsPredicate;
-import org.sonar.iac.docker.checks.utils.command.OptionPredicate;
 import org.sonar.iac.docker.checks.utils.command.PredicateContext;
+import org.sonar.iac.docker.checks.utils.command.PredicateContext.Status;
 import org.sonar.iac.docker.checks.utils.command.SeparatedList;
 import org.sonar.iac.docker.checks.utils.command.SingularPredicate;
 import org.sonar.iac.docker.symbols.ArgumentResolution;
 
 import static org.sonar.iac.docker.checks.utils.ArgumentResolutionSplitter.splitCommands;
-import static org.sonar.iac.docker.checks.utils.command.CommandPredicate.Type.MATCH;
-import static org.sonar.iac.docker.checks.utils.command.CommandPredicate.Type.NO_MATCH;
-import static org.sonar.iac.docker.checks.utils.command.CommandPredicate.Type.OPTIONAL;
-import static org.sonar.iac.docker.checks.utils.command.CommandPredicate.Type.ZERO_OR_MORE;
-import static org.sonar.iac.docker.checks.utils.command.PredicateContext.Status.ABORT;
-import static org.sonar.iac.docker.checks.utils.command.PredicateContext.Status.FOUND_NO_PREDICATE_MATCH;
```

Comment:
There is a limit on a number of static imports in a file

Reply:
Is this really valuable to remove?